src/types/api.ts

```diff
@@ -67,11 +67,65 @@ export type RangeSelector = {
   endOffset: number;
 };
 
+/**
+ * Selector which identifies the Content Document within an EPUB that an
+ * annotation was made in.
+ */
+export type EPUBContentSelector = {
+  type: 'EPUBContentSelector';
+
+  /**
+   * URL of the content document. This can either be an absolute HTTP URL, or
+   * a URL that is relative to the root of the EPUB.
+   */
+  url: string;
```

Comment:
> Can we talk a little about what happens if this is a relative URL/path, as it looks like it might sometimes be?

The main use of the `url` here would be testing if an annotation with a saved `EPUBContentSelector` matches the current chapter, or grouping annotations based on the chapter they were made on. We can use relative URLs in both these contexts, although it is not yet determined what rules we'd apply to compare a relative and an absolute URL. In the case of VitalSource, we could actually resolve the path returned by VitalSource (`/books/...`) into an absolute URL (`https://jigsaw.vitalsource.com/books/...`).There might be cases where it is not possible in general to get an absolute URL for an EPUB Content Document, eg. if a viewer is dynamically extracting the HTML from a `.epub` zip file on the client side, or if viewing an EPUB on the local file system.